project/frontend/src/components/SampleRestaurant.js

```diff
@@ -0,0 +1,360 @@
+/**
+ * A sample restaurant and restaurant detail object for use
+ * with a RestaurantCard component.
+ */
+const restaurant = {
+  hash: 123456789,
+  key: {
+    address: '2162 Larkin St, San Francisco, CA 94109, USA',
+    avgRating: 4.6,
+    id: 'apr75h4bni2pf98h4inujnksjrliu34',
+    latLngCoords: {
+      lat: 37.7979601,
+      lng: -122.4203648,
+    },
+    name: 'Amarena',
+    numRatings: 201,
+    placeTypes: ['Italian', 'Restaurant'],
```

Comment:
Doesn't look like this would affect anything, but right now cuisine type isn't a part of placeTypes. The possible strings in the array are 'meal_delivery', 'meal_takeaway', and 'restaurant'